tensornetwork/linalg/initialization.py

```diff
@@ -105,6 +107,51 @@ def ones(shape: Sequence[int],
   the_tensor = initialize_tensor("ones", shape, backend=backend, dtype=dtype)
   return the_tensor
 
+def ones_like(input: Union[np.ndarray,Tensor],
+         dtype: Optional[Type[np.number]] = None,
+         backend: Optional[Union[Text, AbstractBackend]] = None) -> Tensor:
+  """Return a Tensor shape full of ones the same shape as input
+  Args:
+    input : Object to recieve shape from
+     dtype (optional) : dtype of object
+     backend(optional): The backend or its name."""
+  if backend is None:
+    backend = backend_contextmanager.get_default_backend()
+  elif isinstance(backend, str):
```

Comment:
This elif is unnecessary because the factory will work fine when given a backend object

Reply:
I have replaced the elif and else with just an else to create the backend